tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```
@@ -60,8 +60,9 @@ services:
 {% if cluster_node_count|int > 1 %}
   haproxy:
     image: haproxy
+    user: "1000"
```

This would break if your host UID wasn't 1000. 
```suggestion
    user: "{{ ansible_user_uid }}"
```

---

fixed